app/Models/Product.php

```diff
@@ -32,4 +33,14 @@ public function category()
     {
         return $this->belongsTo(Category::class);
     }
+
+    public function getThumbnailAttribute($value)
+    {
+        return asset('storage/images/'.$value);
+    }
+
+    public function getCreatedAtAttribute($value)
+    {
+        return Carbon::parse($value)->format('d/m/Y');
```

Comment:
cho string vào config

Reply:
suggest change cho mình với

Reply:
```suggestion
        return Carbon::parse($value)->format(config('duong_dan_den_bien_trong_config'));
```
Trong 1 file config: đặt 1 biến có gt 'd/m/Y'